app/assets/stylesheets/application.scss

```
@@ -55,7 +54,10 @@ $sidebar-width: 220px;
 
 #teaser {
   height: $header-height;
-  background: white image-url("teaser.jpg");
+  // Not image-url(): that's a Sprockets/sass-rails Sass function dartsass-rails
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)